### ISSUE TITLE
Input Cursor Bugs When Scrolling

### ISSUE THREAD

**Issue by tsupol** [ionic-team/ionic#12992]
**Ionic version:**  (check one with "x")
(For Ionic 1.x issues, please use https://github.com/ionic-team/ionic-v1)
[ ] **2.x**
[x] **3.x**
[ ] **4.x**

**I'm submitting a ...**  (check one with "x")
[x] bug report
[ ] feature request

**Current behavior:**

The input cursor is out of place, freezing, jiggering and overlapping with header while scrolling. 
See [this video](https://youtu.be/mUzPf1MPpQY) for the issue in action.

**Expected behavior:**

For the input cursor to behave normally.

**Steps to reproduce:**

**Related code:**
Here is the reproducible [repo](https://github.com/tsupol/testForm). This repo is just a sidemenu starting project with inputs copied form the [Input's official doc](https://ionicframework.com/docs/api/components/input/Input/)

**More Info:**
The input cursor issue is also present in emulators as well. 
Moreover, the IOS emulators(tried many) won't show keyboard again after closing it once but this keyboard issue is not present in the device.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

```
cli packages: (/Users/tonsu/.nvm/versions/node/v8.4.0/lib/node_modules)

    @ionic/cli-utils  : 1.12.0
    ionic (Ionic CLI) : 3.12.0

global packages:

    cordova (Cordova CLI) : 7.0.1

local packages:

    @ionic/app-scripts : 2.1.4
    Cordova Platforms  : android 6.2.3 ios 4.4.0
    Ionic Framework    : ionic-angular 3.6.1

System:

    Android SDK Tools : 26.0.2
    ios-deploy        : 1.9.2
    ios-sim           : 6.1.2
    Node              : v8.4.0
    npm               : 5.3.0
    OS                : macOS Sierra
    Xcode             : Xcode 9.0 Build version 9A235

Misc:

    backend : legacy

```

**Comment by kensodemann** [comment]
Hello! Thank you for opening an issue with us!

I was able to duplicate the scrolling / cursor issue with both UIWebView and WKWebView.

I was not able to duplicate the keyboard issue in the emulator at all, however.

Thank you for using Ionic

**Comment by mqasem92** [comment]
I have the same problem

**Comment by ionut-movila** [comment]
Same for me. This look not very nice.

**Comment by SquallX** [comment]
I have same problem only on android

**Comment by mikewagz** [comment]
Also seeing this and getting numerous customer complaints.  All iOS11 so far.  I suspect it has something to do with `viewport-fit=cover` added to the meta viewport tag for iPhoneX support.

**Comment by danielcrk** [comment]
No, this is a bug that has been present on both UIWebview and WKWebview for ages, at least since iOS 7. It is also present in Safari. Has nothing to do with meta tags but with the inertia scrolling on iOS.

**Comment by RedStarThrower** [comment]
Any plans to fix this?

**Comment by Aarbel** [comment]
Up ! did you found a workaround ? Who maintains wkwebview ? Are they Watching community bugs report ?

**Comment by Aarbel** [comment]
Up ! did you found a workaround ? Who maintains wkwebview ? Are they Watching community bugs report ?

**Comment by jasekiw** [comment]
@kensodemann This is a very unappealing bug. Our clients don't want a cursor sliding across the page. I just tested a blank project with a UIWebView and did not reproduce this bug but it is reproduceable in Ionic. I think this bug needs a lot more attention.

**Comment by Ahmed-Masoud** [comment]
Did anyone manage to fix that?

**Comment by Aarbel** [comment]
The problem comes from Webkit and Apple WKWebview. (Webkit is also maintained by Google)
Anyone knows someone in the Webkit team ? 

Links : 
- https://webkit.org/reporting-bugs/
- https://bugs.webkit.org/buglist.cgi?quicksearch=input%20cursor

**Comment by kirtipriya** [comment]
Is there a solution for this issue ?

**Comment by MrDrProfX** [comment]
I see the same issue.
